### PR TITLE
[MaskedTransition] Update Swift example to use MDCFloatingButton

### PR DESCRIPTION
**Pull request by codeman7**
Previously we didn't use MDCFloatingButton or any themers in our example, this adds that.

| Before | After |
| ------- | ------- |
|simulator screen shot - iphone x - 2018-09-06 at 15 32 16|simulator screen shot - iphone x - 2018-09-06 at 15 28 36|

### CODE REVIEW

**Comment by romoore** [comment]
Are those buttons out of the Safe Area? They look like they're bumping right up against the edge of the safe area - can we add a little padding?

File: components/MaskedTransition/examples/MaskedTransitionTypicalUse.swift [hunk 1]
```diff
@@ -169,3 +180,17 @@ private class ModalViewController: UIViewController {
     dismiss(animated: true)
   }
 }
+
+extension MaskedTransitionTypicalUseSwiftExample {
+  // MARK: - CatalogByConvention
+  class func catalogMetadata() -> [String: Any] {
+    return [
+      "breadcrumbs" : [ "Masked Transition", "Masked Transition (Swift)" ],
```
Comment (romoore) [inline components/MaskedTransition/examples/MaskedTransitionTypicalUse.swift:194]: are there no string constants for these?

**Comment by jverkoey** [comment]
Awesome, thank you! I expect this highlights https://github.com/material-components/material-components-ios/issues/2219 as a bug even better now.

File: components/MaskedTransition/examples/MaskedTransitionTypicalUse.swift (hunk 1 above)
Comment (codeman7) [reply components/MaskedTransition/examples/MaskedTransitionTypicalUse.swift:194]: Not that I am aware of.

**Comment by codeman7** [comment]
Addressed the safe area
simulator screen shot - iphone x - 2018-09-06 at 22 46 47

**Review by yarneo — APPROVED** [review]

**Pull request merged**